src/main/java/com/enonic/app/guillotine/graphql/helper/ParamsUrHelper.java

```diff
@@ -0,0 +1,40 @@
+package com.enonic.app.guillotine.graphql.helper;
+
+import java.util.Map;
+import java.util.Objects;
+
+import com.google.common.collect.Multimap;
+
+public class ParamsUrHelper
+{
+    public static void resolveParams( final Multimap<String, String> holder, final Object originalParams )
+    {
+        if ( originalParams instanceof Map )
```

Comment:
what if not? Why we silently skip other types?

Reply:
Yes, the same behavior in XP.
https://github.com/enonic/xp/blob/master/modules/lib/lib-portal/src/main/java/com/enonic/xp/lib/portal/url/AbstractUrlHandler.java#L48
https://github.com/enonic/xp/blob/master/modules/lib/lib-portal/src/main/java/com/enonic/xp/lib/portal/url/AbstractUrlHandler.java#L61